### DISCUSSION TITLE
Is it possible to add a team as reviewers/assignees in GitLab?

### DISCUSSION THREAD

**Discussion by fregdy** [renovatebot/renovate#27669] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

gitlab-runner

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### What is your question?

The [docs](https://docs.renovatebot.com/configuration-options/#reviewers) explain how to achieve this with GitHub but is there a way to do this with GitLab? I saw that this is possible using [CODEOWNERS](https://github.com/renovatebot/renovate/pull/19469) but due to external reasons, I cannot use it for this.

**Comment by rarkins** [comment]
It's not currently possible using the `team:` syntax supported by GitHub. What would it take to add a team in GitLab? e.g. do you assign it to a team name just like you would a regular account, or do you need to extract the members of a team and assign to them individually?

**Reply by fregdy** [reply]
got it, about adding teams, its the second scenario, you need to extract the members and assign them individually.

**Reply by rarkins** [reply]
Can you locate the relevant docs/API endpoints?

**Reply by fregdy** [reply]
About team support from GitLab, I found this [Issue](https://gitlab.com/gitlab-org/gitlab/-/issues/362094), so yes they don't plan on supporting adding team as reviewers, about the docs/endpoint, they are already implemented in Renovate GitLab module [here](https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/gitlab/http.ts#L13) that's why I asked if there was a special syntax like GitHub, or this only applies when using `reviewersFromCodeOwners`?

**Reply by viceice** [reply, marked as answer]
please simply try it out, I think it should work like adding all team members

**Reply by fregdy** [reply]
You are right i tested it out, without `team:` nor `@` like in CODEOWNERS file, and it works, thanks!

**Reply by sass1997** [reply]
Is there a possibility to just allow the default reviewers from the project?
